src/lib/data/audio.ts

```
@@ -148,9 +151,106 @@ export function playStop() {
+        }
+    }
+
+    updateTime();
 }
+
+function getHeadingMarkers() {
+    if (currentAudioPlayer?.headingMarkers) {
+        return currentAudioPlayer.headingMarkers;
+    }
+
+    let headingMarkers = [0.0];
```

ESLint

[error] 'headingMarkers' is never reassigned. Use 'const' instead.

(prefer-const)